packages/run-protocol/src/proposals/demoIssuers.js

```diff
@@ -543,7 +543,7 @@ export const fundAMM = async ({
         );
         assert(secondaryPayment, X`no payment for ${q(issuerName)}`);
 
-        assert(kit.issuer, `No issuer for ${issuerName}`);
+        assert(kit.issuer, X`No issuer for ${issuerName}`);
```

Comment:
```suggestion
        assert(kit.issuer, X`No issuer for ${q(issuerName)}`);
```
I see the assert on line 544 likewise decided to disclose this, which seems reasonable.

Reply:
done